### DISCUSSION TITLE
How do I use Homebrew to upgrade a manually installed App?

### DISCUSSION THREAD

**Discussion by lkxed** [Homebrew/discussions#3103] [Everyday usage] [answered]
For instance, I downloaded an image of Docker from its website which I then installed manually. Now if I want to use Homebrew to upgrade it, is there any way to achieve it (apart from reinstalling it with Homebrew)?

**Comment by SMillerDev** [comment, marked as answer]
No, homebrew can't track software it didn't install. `brew install --force --cask docker` might work to simply replace the existing install though.
